averbis/core/_rest_client.py

```diff
@@ -575,15 +595,53 @@ def __init__(self, project: "Project", name: str):
         self.project = project
         self.name = name
 
+    @property
+    def client(self):
```

Comment:
I'd prefer if the client stays an internal thing to which the objects are connected via a top-level thing (i.e. the Project). It is not something that code using the Process or other classes should have to access directly. Why do you want to expose it on intermediate objects?

Reply:
Alright, will change it.